content/yhdistys/jäsenyys.en.md

```diff
@@ -83,29 +83,65 @@ By showing the **LINKKI MEMBERSHIP STICKER** at purchase, you'll have access to
+Booking method: advance booking only via the Internet: https://twice.shop/laajis/l/laajisvlinehuolto/shop, subcode: HuoltoAJ24, proof of membership still to be provided on the spot.
+
 ### Adventure Park
-* Discount: Adventure Ticket -20% / maximum adventure time 3 hours
-  * Valid every day throughout the adventure season
-  * RESTRICTIONS: Does not apply to the Light Adventure or Quick Adventure
-  * Booking method: Online advance booking only (membership must be proven on-site)
-  * Discount code: **AJPuisto24!**
-
-### Bike Park Downhill Biking
-* Discount: Bike Park Package 50€ with Ghost full-suspension enduro bike / duration 2 hours
-  * Includes lift ticket, rental bike, and protective gear (does not include *keycard ticket base)
-  * Valid every day throughout the bike park season.
-  * Bike model upgrade 15€ (premium models include Commencal, Specialized, Production Privet).
-  * Booking method: Online advance booking only (membership must be proven on-site)
-  * **NOTE!** Contrary to the original instructions, we are currently unable to sell this package online as is. For now, the Bike Park trial package at this special price can be reserved in advance by phone at 0207 436 437 or by email at vuokraamo@laajis.fi
-
-### eFatbike Rental
-* Discount: eFatbike rental 40€ / 2 hours
-  * Includes bike and helmet
-  * Valid every day.
-  * Booking method: Online advance booking only (membership must be proven on-site)
-  * Discount code: **AJeFat23!**
+Discount: adventure ticket 20€ / adventure time max. Max. 3 hours
+Valid on all days throughout the adventure season.
+LIMITATIONS: Not applicable to light adventure
+Booking method: on the spot
+
+### Bikepark downhill cycling
+Discount: Bikepark package 50€ Ghost full suspension enduro bike / duration 2 hours
+Includes lift ticket, rental bike and protective equipment (does not include *keycard)
+Valid on all days throughout the bikepark season.
+Bike model surcharge 15€ (premium models include Commencal, Specialized, Production Privet).
+Booking method: on the spot
+
+### eFatbike rental
+Discount: eFatbike 2 hours rental -15€ off the normal price.
+Includes bike and helmet (does not include insurance).
+Valid on all days.
+Booking method: only advance booking online: https://twice.shop/laajis/l/laajis/shop, subcode: eFat24AJ, proof of membership to be provided on the spot.
+
+The lift ticket/bikepark ticket is loaded onto a Keycard ticket base, which can be reused and remains with the customer. The Keycard base is also valid for several other ski resorts in Finland and Europe. The price of the Keycard ticket base at the time of contract is 7€ (we reserve the right to change the price of the Keycard).
+
+### REDEEMING DISCOUNTS
+If a discount requires a pre-booking, Laajis will provide the subject organisation with a code and a link to redeem the discount. For discounts requiring advance booking, the benefit is not available on the spot but the membership must be identifiable when requested.
```

Comment:
the benefit if not available on the spot -> the benefit is not purchasable(?) on the spot
identifiable -> proven